### PR TITLE
modify aws db snapshot resource

### PR DESCRIPTION
**Pull request by kterada0509**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" comments, they generate extra noise for pull request followers and do not help prioritize the request

References #8958 

Release note for [CHANGELOG](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md):

```release-note
BUG-FIXES:
- resource/aws_db_snapshot: Fix error handling when resource already gone.
ENHANCEMENTS:
- resource/aws_db_snapshot: Modify error message format.
```

Output from acceptance testing:

```
$ make testacc TESTARGS="-run=TestAccAWSDBSnapshot_"
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./... -v -parallel 20 -run=TestAccAWSDBSnapshot_ -timeout 120m
?   	github.com/terraform-providers/terraform-provider-aws	[no test files]
=== RUN   TestAccAWSDBSnapshot_basic
=== PAUSE TestAccAWSDBSnapshot_basic
=== RUN   TestAccAWSDBSnapshot_tags
=== PAUSE TestAccAWSDBSnapshot_tags
=== RUN   TestAccAWSDBSnapshot_disappears
=== PAUSE TestAccAWSDBSnapshot_disappears
=== CONT  TestAccAWSDBSnapshot_basic
=== CONT  TestAccAWSDBSnapshot_disappears
=== CONT  TestAccAWSDBSnapshot_tags
--- PASS: TestAccAWSDBSnapshot_basic (594.11s)
--- PASS: TestAccAWSDBSnapshot_disappears (729.24s)
--- PASS: TestAccAWSDBSnapshot_tags (754.28s)
PASS
ok  	github.com/terraform-providers/terraform-provider-aws/aws	754.398s
```

### CODE REVIEW

File: aws/resource_aws_db_snapshot_test.go [hunk 1]
```diff
@@ -56,6 +58,60 @@ func TestAccAWSDBSnapshot_tags(t *testing.T) {
...
+		}
+
+		request := &rds.DescribeDBSnapshotsInput{
+			DBSnapshotIdentifier: aws.String(rs.Primary.ID),
+		}
+
+		resp, err := conn.DescribeDBSnapshots(request)
+
+		if isAWSErr(err, rds.ErrCodeDBSnapshotNotFoundFault, "") {
+			return nil
```
Comment (bflad) [inline aws/resource_aws_db_snapshot_test.go]:
```suggestion
			continue
```

**Comment by kterada0509** [comment]
Re-run acctest

```
ture/fix-aws_db_snapshot-resource *)$ make testacc TESTARGS="-run=TestAccAWSDBSnapshot_"
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./... -v -parallel 20 -run=TestAccAWSDBSnapshot_ -timeout 120m
?   	github.com/terraform-providers/terraform-provider-aws	[no test files]
=== RUN   TestAccAWSDBSnapshot_basic
=== PAUSE TestAccAWSDBSnapshot_basic
=== RUN   TestAccAWSDBSnapshot_tags
=== PAUSE TestAccAWSDBSnapshot_tags
=== RUN   TestAccAWSDBSnapshot_disappears
=== PAUSE TestAccAWSDBSnapshot_disappears
=== CONT  TestAccAWSDBSnapshot_basic
=== CONT  TestAccAWSDBSnapshot_disappears
=== CONT  TestAccAWSDBSnapshot_tags
--- PASS: TestAccAWSDBSnapshot_basic (606.91s)
--- PASS: TestAccAWSDBSnapshot_tags (673.54s)
--- PASS: TestAccAWSDBSnapshot_disappears (880.67s)
PASS
ok  	github.com/terraform-providers/terraform-provider-aws/aws	880.764s
```

File: aws/resource_aws_db_snapshot_test.go (hunk 1 above)
Comment (kterada0509) [reply aws/resource_aws_db_snapshot_test.go]: Thanks, fixed!

**Comment by bflad** [comment]
This has been released in [version 2.17.0 of the Terraform AWS provider](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#2170-june-28-2019). Please see the [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#provider-versions) or reach out if you need any assistance upgrading.

For further feature requests or bug reports with this functionality, please create a [new GitHub issue](https://github.com/terraform-providers/terraform-provider-aws/issues/new/choose) following the template for triage. Thanks!

**Pull request merged**